src/components/editor-page/renderer-pane/render-iframe.tsx

```diff
@@ -16,7 +16,7 @@ import { ScrollState } from '../synced-scroll/scroll-props'
 import { useOnIframeLoad } from './hooks/use-on-iframe-load'
 import { ShowOnPropChangeImageLightbox } from './show-on-prop-change-image-lightbox'
 
-export interface RenderIframeProps extends RendererProps {
+export interface RenderIframeProps extends Omit<RendererProps, 'baseUrl' | 'disableToc'> {
```

Comment:
I think this could benefit from an explanatory comment…

Reply:
I fixed the types